dev/docker-compose.dev.yml

```diff
@@ -90,7 +90,7 @@ services:
             - cache
     vote:
         build:
-            context: "https://github.com/OpenSlides/openslides-vote-service.git#main"
+            context: "https://github.com/MJJojo97/openslides-vote-service.git#main"
```

Comment:
Is it possible to undo this line and still merge this PR?

Reply:
I removed this change. Can you try whether it is possible to merge it, now? Or what shall I do for testing that?